<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>943</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Stylus Profiler</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonFollowStyle</enum>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="5" column="1">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>&amp;Pressure:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>pressure</cstring>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>&amp;Weight (g):</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>weight</cstring>
          </property>
         </widget>
        </item>
        <item row="6" column="3">
         <widget class="QPushButton" name="add_weight_btn">
          <property name="text">
           <string>&amp;Add</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2" colspan="2">
         <widget class="QLineEdit" name="ts_created">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Stylus id:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>stylus_id</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="2" colspan="2">
         <widget class="QLineEdit" name="ts_updated">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Created on:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>ts_created</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="2" colspan="2">
         <widget class="QLineEdit" name="stylus_id">
          <property name="readOnly">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="2" colspan="2">
         <widget class="QLineEdit" name="pressure"/>
        </item>
        <item row="6" column="2">
         <widget class="QLineEdit" name="weight"/>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Updated on:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>ts_updated</cstring>
          </property>
         </widget>
        </item>
        <item row="7" column="0" colspan="4">
         <widget class="TableWidget" name="data">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <property name="columnCount">
           <number>2</number>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>130</number>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <column/>
          <column/>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Operator:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>operator_id</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="2" colspan="2">
         <widget class="QLineEdit" name="operator_id"/>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Tablet id:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>tablet_id</cstring>
          </property>
         </widget>
        </item>
        <item row="4" column="2" colspan="2">
         <widget class="QLineEdit" name="tablet_id"/>
        </item>
       </layout>
      </widget>
      <widget class="PlotWidget" name="view">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Main toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionExit"/>
   <addaction name="separator"/>
   <addaction name="actionFullScreen"/>
   <addaction name="actionInfo"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normaloff>../icons/24x24/open.png</normaloff>../icons/24x24/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset>
     <normaloff>../icons/24x24/exit.png</normaloff>../icons/24x24/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="actionInfo">
   <property name="icon">
    <iconset>
     <normaloff>../icons/24x24/info.png</normaloff>../icons/24x24/info.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset>
     <normaloff>../icons/24x24/new.png</normaloff>../icons/24x24/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normaloff>../icons/24x24/save.png</normaloff>../icons/24x24/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionFullScreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icons/24x24/fullscreen.png</normaloff>../icons/24x24/fullscreen.png</iconset>
   </property>
   <property name="text">
    <string>Toggle Full Screen</string>
   </property>
   <property name="toolTip">
    <string>Toggle Full Screen</string>
   </property>
   <property name="shortcut">
    <string>F11</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
  <customwidget>
   <class>TableWidget</class>
   <extends>QTableWidget</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>ts_created</tabstop>
  <tabstop>ts_updated</tabstop>
  <tabstop>operator_id</tabstop>
  <tabstop>stylus_id</tabstop>
  <tabstop>tablet_id</tabstop>
  <tabstop>pressure</tabstop>
  <tabstop>weight</tabstop>
  <tabstop>add_weight_btn</tabstop>
  <tabstop>data</tabstop>
  <tabstop>view</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>weight</sender>
   <signal>returnPressed()</signal>
   <receiver>add_weight_btn</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>221</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pressure</sender>
   <signal>returnPressed()</signal>
   <receiver>weight</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>312</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>operator_id</sender>
   <signal>returnPressed()</signal>
   <receiver>stylus_id</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>255</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stylus_id</sender>
   <signal>returnPressed()</signal>
   <receiver>pressure</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
